R/doAzureParallel.R

```diff
@@ -311,35 +311,35 @@ setVerbose <- function(value = FALSE) {
       resourceFiles <-
         append(resourceFiles, requiredJobResourceFiles)
 
-      cloudCombineEnabled <- list(name = "cloudCombineEnabled", value = "TRUE")
+      cloudCombineEnabled <-
+        list(name = "cloudCombineEnabled", value = "TRUE")
 
       chunkSize <- 1
-      
-      if(!is.null(obj$options$azure$chunkSize)){
-        chunkSize <- obj$options$azure$chunkSize
-      }
-      
-      if(!is.null(obj$options$azure$chunksize)){
```

Comment:
Remove `if(!is.null(obj$options$azure$chunksize)){` and keep `if(!is.null(obj$options$azure$chunkSize)){` since all our documentations uses chunkSize